Data/Scripts/001_Technical/002_RubyUtilities.rb

```diff
@@ -162,6 +159,147 @@ def self.move(source, destination)
   end
 end
 
+#===============================================================================
+#  Extensions for the `Color` class
+#===============================================================================
+class Color
+  # alias for old constructor
+  alias init_original initialize unless self.private_method_defined?(:init_original)
+  #-----------------------------------------------------------------------------
+  # new constructor accepts RGB values as well as a hex number or string value
+  #-----------------------------------------------------------------------------
+  def initialize(*args)
+  	pbPrintException("Wrong number of arguments! At least 1 is needed!") if args.length < 1
+  	if args.length == 1
+      if args.first.is_a?(Fixnum)
+        hex = args.first.to_s(16)
+      elsif args.first.is_a?(String)
+        try_rgb_format = args.first.split(',')
+        return init_original(*try_rgb_format) if try_rgb_format.length.between?(3, 4)
```

Comment:
Isn't this passing three/four strings to `init_original` when it should be passing three/four integers?

Reply:
Yeah, I forgot the `.map(&:to_i)` there.